src/main/java/com/exasol/dbbuilder/User.java

```diff
@@ -0,0 +1,109 @@
+package com.exasol.dbbuilder;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import com.exasol.dbbuilder.objectwriter.DatabaseObjectWriter;
+
+/**
+ * Database user
+ */
+public class User extends AbstractDatabaseObject {
+    private final String password;
+    private final Map<DatabaseObject, ObjectPrivilege[]> objectPrivileges = new HashMap<>();
+    private final Set<SystemPrivilege> systemPrivileges = new HashSet<>();
+
+    /**
+     * Create a new database user.
+     *
+     * @param writer database object writer
+     * @param name   user name
+     */
+    public User(final DatabaseObjectWriter writer, final String name) {
+        super(writer, name);
+        this.password = name + "PWD";
```

Comment:
Add also another constructor for user provided password.

Reply:
Added constructor and unit test.
Also added new methods in `DatabaseObjectFactory` and `ExasolObjectFactory`.